src/oaklib/implementations/funowl/funowl_implementation.py

```diff
@@ -97,7 +97,7 @@ def label(self, curie: CURIE, lang: Optional[LANGUAGE_TAG] = None) -> str:
             else:
                 raise ValueError(f"Label must be literal, not {label}")
 
-    def entities(self, filter_obsoletes=True, owl_type=None) -> Iterable[CURIE]:
+    def entities(self, filter_obsoletes=True, owl_type=None) -> Iterable[Union[URI, CURIE]]:
```

Comment:
## Order of types?
This is a super minor thing, but I am going to guess that if there is a Python convention, maybe it should be `Union[CURIE, URI]` for alphabetical reasons, but I hose `Union[URI, CURIE]` because that's usually the order we speak them in the semantic web world.

Reply:
Does not matter.